src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -145,6 +149,9 @@ public synchronized void launch(SlaveComputer computer, TaskListener listener) {
                     .orElse(null);
             node.setNamespace(namespace);
 
+            // register a namespace informer (if not registered yet) to show relevant pod events in build logs
+            cloud.registerPodInformer(node, client, namespace);
```

Comment:
What would happen on controller restart ? Do we care about re-attaching existing nodes or do we only care about new nodes?

Reply:
Good point. I think once the pod is running and containers ready, these pod events are less interesting (actually there are usually no events until the pod is deleted).
IMO we can live without this, until it proves to be needed.